File: sketches/sketches.model/src/main/java/edu/kit/kastel/lissa/sketches/model/impl/elements/uml/UMLInterface.java
package edu.kit.kastel.lissa.sketches.model.impl.elements.uml;

import edu.kit.kastel.lissa.sketches.model.elements.uml.IUMLInterface;
import edu.kit.kastel.lissa.sketches.model.types.SketchBoxTypes;

public class UMLInterface extends UMLThing implements IUMLInterface {
	private static final long serialVersionUID = 3310996720085779684L;

	public UMLInterface() {
		// NOP
	}

	@Override
	public SketchBoxTypes getCurrentInterpretation() {
		return SketchBoxTypes.INTERFACE;
	}

}

File: sketches/model/src/main/kotlin/edu/kit/kastel/lissa/sketches/model/impl/elements/uml/UMLInterface.kt
package edu.kit.kastel.lissa.sketches.model.impl.elements.uml

import edu.kit.kastel.lissa.sketches.model.elements.uml.IUMLInterface
import edu.kit.kastel.lissa.sketches.model.types.SketchBoxTypes

class UMLInterface : UMLThing(), IUMLInterface {
    override fun currentInterpretation() = SketchBoxTypes.INTERFACE
}
